ext/gmp/tests/gmp_ecc_cryptography.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+test some of the simple operations done in ECC cryptography (GH-16870)
+--EXTENSIONS--
+gmp
+--FILE--
+<?php
+$big_128 = gmp_init("275458973678998164972199298356823445708", 10);
+$big_256 = gmp_init("105897378200704506708747774951422238999742054168150946506894304230846229930690", 10);
+$big_384 = gmp_init("6416793922163250205142333121845147158821454739198730072980369042145699078647151545580979451456624745026383368515606", 10);
+$big_521 = gmp_init("1780138296718969229154737597263093326930734138014367844231086609903991655888322554437298817404082596582997382075361284417827317144352840347188199593652743789", 10);
+// in ecc crypto key lengths of 256, 384 and 521 are typical,
```

Comment:
```suggestion
// in ecc crypto key lengths of 256, 384 and 521 (yes, 521, not 512) are typical,
```

This is a very common source of confusion and we don't want someone “helpfully” fixing the typo.

Reply:
suggestion applied.